pkgs/top-level/all-packages.nix

```diff
@@ -3144,7 +3144,16 @@ let
 
   vpnc = callPackage ../tools/networking/vpnc { };
 
-  openconnect = callPackage ../tools/networking/openconnect.nix { };
+  openconnect = openconnect_openssl;
+
+  openconnect_openssl = callPackage ../tools/networking/openconnect.nix {
+    gnutls = null;
+  };
+
+  openconnect_gnutls = lowPrio (openconnect.override {
+    openssl = null;
+    gnutls = gnutls;
```

Comment:
Just a nitpick: it's more typical to write ``inherit gnutls;`` instead, but it doesn't matter...